src/use_event_listener/index.jsx

```diff
@@ -1,5 +1,13 @@
 import { useEffect, useRef } from 'react';
 
+/**
+ * @param ref
+ * @param type
+ * @param handler
+ * @param passive
+ * @param capture
+ * @param once
+ */
 export const useEventListener = (
```

Comment:
any idea why it doesn't get detected as a function? 

https://goodhood-eu.github.io/nebenan-react-hocs/global.html#useEventListener

Reply:
ES6 \ classes should be declared with `@function` (done)